gateway-ha/src/test/java/io/trino/gateway/ha/security/TestLbAuthenticator.java

```diff
@@ -107,13 +107,10 @@ public void testAuthenticatorMissingClaim()
     public void testPresetUsers()
             throws Exception
     {
-        Map presetUsers = new HashMap<String, UserConfiguration>()
-        {
-            {
-                this.put("user1", new UserConfiguration("priv1, priv2", "pass1"));
-                this.put("user2", new UserConfiguration("priv2, priv2", "pass2"));
-            }
-        };
+        Map presetUsers = ImmutableMap.of("user1",
+                new UserConfiguration("priv1, priv2", "pass1"),
+                "user2",
+                new UserConfiguration("priv2, priv2", "pass2"));
```

Comment:
nit: Avoid raw usage of parameterized map and wrap per entry:
```java
        Map<String, UserConfiguration> presetUsers = ImmutableMap.of(
                "user1", new UserConfiguration("priv1, priv2", "pass1"),
                "user2", new UserConfiguration("priv2, priv2", "pass2"));
```
Same for another change.